modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/user/approval/v1/UserMeApprovalTest.java

```diff
@@ -117,7 +117,7 @@ public void testListTasksWhenEmpty() {
     public void testListTasksWhenAvailable() throws Exception {
 
         addAssociationForMatch();
-        for (int i = 1; i <= 3; i++) {
+        for (int i = 1; i <= 6; i++) {
```

Comment:
Can we get this to a constant?

Reply:
Done please add your comments on constant name